core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/RJSElectricalProfile.java

```diff
@@ -0,0 +1,23 @@
+package fr.sncf.osrd.railjson.schema;
+
+import com.squareup.moshi.Json;
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.Moshi;
+import com.squareup.moshi.Types;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+import fr.sncf.osrd.railjson.schema.infra.trackranges.RJSTrackRange;
+
+public class RJSElectricalProfile {
+    static Type listOfThis = Types.newParameterizedType(List.class, RJSElectricalProfile.class);
+    public static final JsonAdapter<List<RJSElectricalProfile>> list_adapter = new Moshi.Builder().build().adapter(listOfThis);
+
```

Comment:
please move this to where you use it (you can keep it static)
please also rename list_adapter to profileListAdapter

Reply:
I don't understand what you mean by move it to where I use it.
In all other RJS classes, the `adapter` is defined inside the class.

Reply:
alright, fine